Apps/frontend/src/views/ControlPanelBottom.svelte

```diff
@@ -33,11 +41,11 @@
       Channel
     {/if}
   </th>
-  <th>Start/Stop</th>
-  <th>Thickness</th>
-  <th>Offset</th>
-  <th>Time Sweep</th>
-  <th>Amplitude</th>
+  <th><ControlColumnHeader label="Start/Stop" icon="toggle" tooltip={TOOLTIP_CONTROL_HEADER_ONOFF} /></th>
```

Comment:
We could think about introducing labels for each header text as well. Doesn't need to be in your issue, but what do you think about it?